source/simulator/assembly.cc

```diff
@@ -1140,6 +1140,7 @@ namespace aspect
 
     const UpdateFlags update_flags = update_values |
                                      update_gradients |
+                                     update_hessians |
```

Comment:
Can we make this dependent on using SUPG? This will make assembly more expensive.